app/services/storage.js

```diff
@@ -94,14 +111,20 @@ export default Ember.Service.extend(Ember.Evented, {
   },
 
   setupRemoteStorage() {
+    const remoteStorage = this.get('remoteStorage');
```

Comment:
I think there's a reason for that, because when you move it, the initial render jumps from disconnected to connected.